substrate/frame/support/src/traits/schedule.rs

```diff
@@ -130,7 +130,8 @@ impl<T: Decode, H> MaybeHashed<T, H> {
 	}
 }
 
-// TODO: deprecate
+#[allow(deprecated)]
+#[deprecated = "traits::schedule::v1 is deprecated. Please use v3 instead."]
```

Comment:
There is some uniform grammar for this:  
https://github.com/paritytech/polkadot-sdk/blob/6b5995ffd162d88df1e3baeaafe05869d1a7d966/docs/contributor/DEPRECATION_CHECKLIST.md#L25